exrsplit/__main__.py

```diff
@@ -142,9 +149,32 @@ def split_exr(args):
             exr.close()
 
 
+def list_exr(args):
+    for inputfile in args.image:
+        exr = _open_inputfile(inputfile)
+        try:
+            header = exr.header()
+            if args.layer is not None:
+                channels = [exrsplit.EXRChannel(header, layer) for layer in header['channels']
+                            if layer.startswith(tuple(args.layer))]
+            else:
+                channels = [exrsplit.EXRChannel(header, layer) for layer in header['channels']]
+            grouped_channels = exrsplit.group_channels(channels)
+            for layer_i, layer in enumerate(grouped_channels):
+                layer_channels = ",".join(sorted([str(x.channel) for x in layer]))
+                print('{}/{} - Layer: {}, Channels: ({})'.format(layer_i + 1,
+                                                                 len(grouped_channels),
+                                                                 layer[0].layer,
+                                                                 layer_channels))
+        finally:
+            exr.close()
+
+
 def main(args):
     if args.merge:
         merge_exr(args)
+    elif args.list:
```

Comment:
I think list should have a higher priority (if the user gives `-l`, only list and exit). Can you put it on top of merge?